code/game/gamemodes/objectives_subtypes.dm

```diff
@@ -0,0 +1,58 @@
+/datum/objective/debrain/command
+	name = "Debrain command"
+	target_jobs = list("Captain", "Head of Personnel", "Head of Security", "Research Director", "Chief Engineer", "Chief Medical Officer", "Quartermaster")
+
+/datum/objective/debrain/science
+	name = "Debrain science"
+	target_jobs = list("Research Director", "Scientist", "Roboticist", "Geneticist")
+
+/datum/objective/assassinate/mindshielded
+	name = "Assassinate mindshielded"
+	mindshielded_target = TRUE
+
+/datum/objective/assassinate/nomindshield
+	name = "Assassinate non-mindshielded"
+	not_mindshielded_target = TRUE
+
+/datum/objective/assassinate/syndicate
+	name = "Assassinate syndicate agent"
+	syndicate_target = TRUE
+
+/datum/objective/assassinate/syndicate/update_explanation_text()
+	..()
+	if(target?.current)
+		explanation_text = "Assassinate [target.current.real_name], the Syndicate agent undercover as the [target.assigned_role]."
+
+/datum/objective/assassinate/medical
+	name = "Assassinate medical"
+	target_jobs = list("Chief Medical Officer", "Medical Doctor", "Paramedic", "Chemist", "Virologist", "Psychologist", "Coroner")
+
+/datum/objective/assassinate/engineering
+	name = "Assassinate engineering"
+	target_jobs = list("Chief Engineer", "Station Engineer", "Atmospherics Technician")
+
+/datum/objective/assassinateonce/animal_abuser
+	name = "Assassinate animal abuser once"
+	target_jobs = list("Head of Personnel", "Quartermaster", "Cargo Technician", "Bartender", "Chef", "Botanist", "Geneticist", "Virologist")
+
+/datum/objective/assassinateonce/animal_abuser/update_explanation_text()
+	..()
+	if(target?.current)
+		explanation_text = "Teach [target.current.real_name], the animal abuser and alleged [target.assigned_role], a lesson they will not forget. The target only needs to die once for success."
```

Comment:
Personal opinion but I really don't like this or think we should have this as an objective.

Reply:
Is the issue a gameplay one, or the subject matter?

Reply:
IMO subject matter. We've removed similar abductor objectives iirc

Reply:
Yeah, I think this could probably be reworded if nothing else; calling people's characters animal abusers is a little icky.

Reply:
Removed 'abuser' vocabulary from the code, description and the objective explanation. Now looks like a regular teach a lesson objective, still targeting the same jobs.

Reply:
It makes sense that it's written from the perspective of a PETA like org, but yeah like I wouldn't want my own characters characterized like that, nor have someone take the objective a bit too seriously and start screaming about that ic.